modules/private_link/main.tf

```diff
@@ -0,0 +1,66 @@
+data "kubernetes_ingress_v1" "ingress" {
+  metadata {
+    name = var.ingress_name
+  }
+}
+
+locals {
+  lb_name = data.kubernetes_ingress_v1.ingress.metadata[0].annotations != null ? data.kubernetes_ingress_v1.ingress.metadata[0].annotations["ingress.kubernetes.io/forwarding-rule"] : ""
```

Comment:
`lb_name` but you are looking at the forward rulename?
Is this value set by the ingress controller when provisioning?

Reply:
This is not being tested ... It will be tested after helm version upgrade to 1.13

Reply:
Okay I approved the helm PR. I would perf not to use data type and pass in the name explicitly

Reply:
So lets make sure to test it with the new helm chart before we merge it

Reply:
@jsbroks  ingress doesn't have any annotation that allows you specify the loadbalancer name that will be created by the gce controller .
And we need the name to link the internal load balancer to the private service .
Checked the code also there is no such supported annotation https://github.com/kubernetes/ingress-gce/blob/master/pkg/annotations/ingress.go